Tests/Repository/ServiceEntityRepositoryTest.php

```diff
@@ -33,4 +35,15 @@ public function testConstructorThrowsExceptionWhenNoManagerFound(): void
         $repo = new ServiceEntityRepository($registry, TestEntity::class);
         $repo->getClassName();
     }
+
+    /** @requires function \Symfony\Component\VarExporter\ProxyHelper::generateLazyGhost */
+    public function testConstructInitializesWhenImplementingLazyObjectInterface(): void
+    {
+        $registry = $this->getMockBuilder(ManagerRegistry::class)->getMock();
+        $this->expectException(LogicException::class);
+
+        new class ($registry, TestEntity::class) extends ServiceEntityRepository implements LazyObjectInterface {
```

Comment:
You can copy line 34 here I think, and add another suppress annotation for `MissingDependency`, since Psalm runs with 7.4 (I'm assuming there is no way to get `LazyObjectInterface` for that version of PHP)

Reply:
that did it, thanks. Remaining failures are unrelated.